versioned/storage/common-proto/src/main/proto/org/projectnessie/versioned/storage/common/proto/storage.proto

```diff
@@ -90,7 +91,6 @@ enum CompressionProto {
 }
 
 message TagProto {
-  bytes commit_id = 1;
```

Comment:
Isn't this a breaking change? How can we deserialize an existing tag that has this field?

Reply:
Also you might want to add `reserved 1;` and document why the field is gone.

Reply:
The tag-objects have never been used. So this breaking change is fine.

Reply:
OK. In this case you could even renumber the fields starting from 1.